runhouse/main.py

```diff
@@ -43,7 +44,7 @@
 
 # For printing with typer
 console = Console()
-logger = logging.getLogger(__name__)
+from runhouse.logger import logger
```

Comment:
i think we can even simplify this further to: `from runhouse import logger`

if we add `from .logger import logger` to the main `__init__`